core/models.py

```diff
@@ -1 +1,16 @@
 from django.db import models
+from accounts.models import User
+
+
+class Event(models.Model):
+    pass
+
+
+class Comment(models.Model):
+    name = models.ForeignKey(User, on_delete=models.CASCADE)
+    event = models.ForeignKey(Event, on_delete=models.CASCADE)
+    description = models.TextField(max_length=200, null=False, blank=True)
```

Comment:
Please change 'description' to 'comment' and max_length to 500 as in diagram. I also suggest to change null to True as this will be less risky for dependencies between User and Event relationship when creating an object in DB.